numba/targets/arraymath.py

```diff
@@ -3541,6 +3542,32 @@ def np_extract_impl(condition, arr):
 
     return np_extract_impl
 
+  
```

Comment:
Is there some extra whitespace on this line?

Reply:
whitespace is like sand at a beach house :-) I deleted the lines and created them again, hopefully it's gone. If not I'll do a flake8 fix at the end.